spec/javascripts/ChangePassword/ChangePassword.spec.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import React from 'react'
+import { mount } from 'enzyme'
+import {ChangePassword} from 'ChangePassword'
```

Comment:
Let's be consistent with the curly space rule: either add a space or don't.
```suggestion
import { ChangePassword } from 'ChangePassword'
```